tests/spread/general/extension-flask/rockcraft.yaml

```diff
@@ -0,0 +1,9 @@
+name: flask-extension
+summary: OCI image for a flask project.
+description: OCI image for a flask project.
+version: "0.1"
+base: bare
+license: Apache-2.0
+
+extensions:
```

Comment:
we need this entry at least referenced now that we are referring to one

https://github.com/canonical/rockcraft/blob/main/docs/reference/rockcraft.yaml.rst

Reply:
Added, thanks!